Hardware/Hardware/CardReader/PaymentIntent.swift

```diff
@@ -106,7 +111,8 @@ public extension PaymentIntent {
                           siteURL: String? = nil,
                           orderID: Int64? = nil,
                           orderKey: String? = nil,
-                          paymentType: PaymentTypes? = nil
+                          paymentType: PaymentTypes? = nil,
```

Comment:
nit;

There's broken alignment for the rest of the parameters

Reply:
Updated the indentation of these misaligned parameters in https://github.com/woocommerce/woocommerce-ios/pull/14479/commits/b601a7c11a25b0dc01b4655393b1b0493fbd8939